bsp_diff/celadon_ivi/kernel/linux-intel-lts2021/0002-drm-bridge-Add-max-serdes-deserdes-driver.patch

```diff
@@ -0,0 +1,1511 @@
+From 2f7d3086ac68c220240c5742959f71ec5eda9584 Mon Sep 17 00:00:00 2001
+From: "Jia, Lin A" <lin.a.jia@intel.com>
+Date: Thu, 6 Jul 2023 14:16:20 +0800
+Subject: [PATCH] drm/bridge: Add max serdes/deserdes driver
+
+Enable max dp serdes/deserdes function for 1080p and 2k resolution
+---
+ drivers/gpu/drm/bridge/Makefile          |   1 +
+ drivers/gpu/drm/bridge/max/Kconfig       |   8 +
+ drivers/gpu/drm/bridge/max/Makefile      |   3 +
+ drivers/gpu/drm/bridge/max/max_ser_drv.c | 576 ++++++++++++++++++++++
+ drivers/gpu/drm/bridge/max/max_ser_drv.h | 114 +++++
+ drivers/gpu/drm/i915/Makefile            |   3 +-
+ drivers/gpu/drm/i915/i915_dp_ser_drv.c   | 590 +++++++++++++++++++++++
+ drivers/gpu/drm/i915/i915_dp_ser_drv.h   | 113 +++++
+ drivers/gpu/drm/i915/i915_module.c       |   3 +
+ 9 files changed, 1410 insertions(+), 1 deletion(-)
+ create mode 100755 drivers/gpu/drm/bridge/max/Kconfig
+ create mode 100755 drivers/gpu/drm/bridge/max/Makefile
+ create mode 100755 drivers/gpu/drm/bridge/max/max_ser_drv.c
+ create mode 100755 drivers/gpu/drm/bridge/max/max_ser_drv.h
+ create mode 100755 drivers/gpu/drm/i915/i915_dp_ser_drv.c
+ create mode 100755 drivers/gpu/drm/i915/i915_dp_ser_drv.h
+
```

Comment:
If the kernel module is still needed for developing purpose, please separate it into another pr.